modules/oadp-support-backup-data-immutability.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/installing/about-installing-oadp.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="oadp-support-backup-data-immutability_{context}"]
+= OADP does not support backup data immutability
+
+Starting with {oadp-short} 1.3, backups may not function as expected when the target object storage has an immutability option configured. These immutability options are referred to by different names, for example:
+
+* Object lock
+* Object retention
+* Bucket versioning
+* Write Once Read Many (WORM) buckets
+
+The primary reason for the absence of support is that {oadp-short} initially saves the state of a backup as _finalizing_ and then scrutinizes whether any asynchronous operations are in progress. If there are asynchronous operations in progress, {oadp-short} needs to wait for all of them to be finished before moving the backup state to _complete_. If there are no asynchronous  operations, and the state is moved to _complete_ immediately. Regardless, {oadp-short} must modify the metadata in object storage, which is not be possible if an immutability option is configured on the object storage.
+
+With versions before {oadp-short} 1.3, an object storage with an immutability configuration was not supported. You might see some problems even though backups are working, for example, version objects are not deleted when backup is deleted.
+
+[NOTE]
+====
+Depending on the specific provider and configuration, backups might work in some cases.
+====
+
+* AWS S3 service supports backups because S3 object lock only applies to versioned buckets, and you can still update the object data for the new version. However, when backups are deleted, old versions of the objects are not deleted.
```

Comment:
```suggestion
* AWS S3 service supports backups because an S3 object lock only applies to versioned buckets. You can still update the object data for the new version. However, when backups are deleted, old versions of the objects are not deleted.
```